cluster-install/install.sh

```diff
@@ -62,7 +64,7 @@ readonly __COMPONENTS_VALID_VALUES=("$__COMPONENTS_ALL_ASTRA_CONTROL" "$__COMPON
 readonly __DEFAULT_DOCKER_HUB_IMAGE_REGISTRY="docker.io"
 readonly __DEFAULT_DOCKER_HUB_IMAGE_BASE_REPO="netapp"
 readonly __DEFAULT_ASTRA_IMAGE_REGISTRY="cr.astra.netapp.io"
-readonly __DEFAULT_IMAGE_TAG="$__RELEASE_VERSION"
+readonly __DEFAULT_ASTRA_IMAGE_BASE_REPO=""
```

Comment:
Seems silly to have a default constant set to empty, but it's used in 2 places and I didn't want those to get unsynchronized in case we ever change this value.